.changeset/tidy-llamas-deliver.md

```diff
@@ -0,0 +1,8 @@
+---
+"@commercetools-frontend/application-shell": patch
+---
+
+refactor(test-utils): to use flopflip test provider
+
+In this release we use the `TestProviderFlopFlip` when using our `test-utils` or the `SetupFlipFlip` component. 
+This change just not affect consumers of those packages. You can remove any `adapter` you may have passed to any `react-testing-library` renderer. These are not needed anymore.
```

Comment:
```suggestion
When using `@commercetools-frontend/application-shell/test-utils`, we now render a `TestProviderFlopFlip` instead of the normal `ConfigureFlopFlip` with the `memory` adapter.<br/>
This change simplifies how feature flags are propagated during tests and should not affect the usage of the test-utils.

> In the very unlikely case you have been passing the `adapter` option to the test-utils, you can remove that as it's not necessary anymore.
```